tests/approaches/test_gnn_metacontroller_approach.py

```diff
@@ -77,11 +75,11 @@ def test_gnn_metacontroller_approach_with_envs(env_name, num_epochs):
         policy = approach.solve(task, timeout=CFG.timeout)
         with pytest.raises(ApproachFailure) as e:
             policy(task.init)
-        assert "GNN metacontroller could not sample an option" in str(e)
+        assert "Metacontroller could not sample an option" in str(e)
```

Comment:
do we need a separate test file for the new file? or no bc its abstract?

Reply:
yeah nah